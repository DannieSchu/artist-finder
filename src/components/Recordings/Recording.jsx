import React from 'react';
import PropTypes from 'prop-types';
import styles from '../../global-styles/item.css';

const Recording = ({ title }) => (
  <section className={styles.item}>
    <h3>{title}</h3>
  </section>
);

Recording.propTypes = {
  title: PropTypes.string.isRequired
};

export default Recording;
