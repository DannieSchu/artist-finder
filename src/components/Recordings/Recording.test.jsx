import React from 'react';
import { shallow } from 'enzyme';
import Recording from './Recording.jsx';

describe('Recording', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(
      <Recording
        title="Wonderful World"
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
