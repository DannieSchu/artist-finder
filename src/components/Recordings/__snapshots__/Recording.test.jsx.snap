// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recording matches a snapshot 1`] = `
<section
  className="item"
>
  <h3>
    Wonderful World
  </h3>
</section>
`;
